CHANGELOG.latest.md

```diff
@@ -1,4 +1,58 @@
-- Updates log message for `createAlias` to improve clarity
-    https://github.com/RevenueCat/purchases-ios/pull/498
-- Adds `rc_` to all Foundation extensions to prevent name collisions
-    https://github.com/RevenueCat/purchases-ios/pull/500
+
+### Identity V3:
+
+#### New methods
+- Introduces `logIn`, a new way of identifying users, which also returns whether a new user has been registered in the system. 
+`logIn` uses a new backend endpoint. 
+- Introduces `logOut`, a replacement for `reset`. 
+
+#### Deprecations
+- deprecates `createAlias` in favor of `logIn`
+- deprecates `identify` in favor of `logIn`
+- deprecates `reset` in favor of `logOut`
+- deprecates `allowSharingAppStoreAccount` in favor of dashboard-side configuration
+
+    https://github.com/RevenueCat/purchases-ios/pull/453
+    https://github.com/RevenueCat/purchases-ios/pull/438
+    https://github.com/RevenueCat/purchases-ios/pull/506
+
+
+### Other changes: 
+
+#### Public additions
+##### SharedPurchases nullability
+- Fixed `sharedPurchases` nullability
+- Introduced new property, `isConfigured`, that can be used to check whether the SDK has been configured and `sharedPurchases` won't be `nil`.
+    https://github.com/RevenueCat/purchases-ios/pull/508
+
+##### Improved log handling
+- Added new property `logLevel`, which provides more granular settings for the log level. Valid values are `debug`, `info`, `warn` and `error`.
+- Added new method, `setLogHandler`, which allows developers to use their own code to handle logging, and integrate their existing systems.
+    https://github.com/RevenueCat/purchases-ios/pull/481
+    https://github.com/RevenueCat/purchases-ios/pull/515
+
+
+#### Deprecations
+- Deprecated `debugLogsEnabled` property in favor of `LogLevel`. Use `Purchases.logLevel = .debug` as a replacement.
```

Comment:
we should update the instructions at this link: https://docs.revenuecat.com/docs/debugging